flatdata-cpp/test/MultiVectorTest.cpp

```diff
@@ -53,6 +53,32 @@ static auto create_view_with_3_items = []( )
     return std::make_pair( std::move( storage ), std::move( view ) );
 };
 
+static const size_t NUM_ITEM_TO_FLUSH = 32 * 1024 * 1024 / 4 + 1024; // enough to flush, and them some
+
+static auto create_storage_with_enough_items_to_flush = []( )
+{
+    auto storage = MemoryResourceStorage::create( );
+    auto vector = storage->create_multi_vector< TestIndexType48, AStruct, BStructMutator, CStruct >(
```

Comment:
Why `BStructMutator` and not `BStruct`?

Reply:
Done. This view does not need to test that Mutator/Accessor are the same for MultiVector